### PR TITLE
[Documentation] Pristine documentation

### PR DESCRIPTION
**Pull request by denniss**

### CODE REVIEW

File: man/bundle-pristine.ronn [hunk 1]
```diff
@@ -0,0 +1,13 @@
+bundle-pristine(1) -- Restores installed gems to its pristine condition
```
Comment (colby-swandale) [inline man/bundle-pristine.ronn]: i would prefer `to their pristine condition`

File: man/bundle-pristine.ronn [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+===========================================================================
+
+## SYNOPSIS
+
+`bundle pristine`
+
+## DESCRIPTION
+
+Restores installed gems to its pristine condition.
```
Comment (colby-swandale) [inline man/bundle-pristine.ronn]: same as above regarding `their`

Comment (colby-swandale) [inline man/bundle-pristine.ronn]: It feels weird reading 4 statements like this out loud. We can blend some these sentences together?  How does something like:

```
`pristine` restores the installed gems in the bundle to their pristine condition using the local gem
cache from RubyGems. For git gems - a forced checkout will be performed.

Note the Bundler gem cannot be restored to its original state with `pristine`. 

```

sound?

Comment (denniss) [inline man/bundle-pristine.ronn]: Much better.

**Comment by colby-swandale** [comment]
thank you! @bundlerbot r+

**Pull request merged**